matchms/filtering/remove_peaks_outside_top_k.py

```diff
@@ -0,0 +1,40 @@
+import numpy
+from ..Spikes import Spikes
+from ..typing import SpectrumType
+
+
+def remove_peaks_outside_top_k(spectrum: SpectrumType, k: int = 6,
+                               mz_window: float = 50) -> SpectrumType:
+
+    """Keep peaks only if they are in the top k intense peaks
+       within a desired mz (Da) window.
```

Comment:
Dont understand what the mz window does. 
1. If one top peak has a peak (outside top k) near it within the mz window will it be kept or removed?
2. If 2 top k peaks are within the window will one be discarded?

Reply:
@kiaashour can you add a bit more description to the docstring?

To Stefans questions: 1. that will be kept. 2. no, all top k peaks are kept.

My docstring suggestion: ``"""Remove all peaks which are not within *mz_window* of at least one of the *k* highest intensity peaks of the spectrum.``